frontend/assets/css/colors.scss

```diff
@@ -124,6 +127,19 @@
 
   --toggle-button-slider-color: var(--light-grey);
 
+  --toast-message-success-color: var(--green);
+  --toast-background-success-color: var(--mint-green);
+  --toast-border-sucess-color: var(--green);
+
+  --toast-message-info-color: var(--light-blue);
+  --toast-background-info-color: var(--sky-blue);
+  --toast-border-info-color: var(--light-blue);
+
+  --toast-message-error-color: var(--flashy-red);
+  --toast-background-error-color: var(--pastel-red);
+  --toast-border-error-color: var(--flashy-red);
+
+
   &.dark-mode {
```

Comment:
might there be there different versions for `light` and `dark mode`?

Reply:
Not that i am aware of that.